receiver/httpcheckreceiver/config.go

```diff
@@ -24,13 +24,17 @@ var (
 // Config defines the configuration for the various elements of the receiver agent.
 type Config struct {
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
-	confighttp.HTTPClientSettings           `mapstructure:",squash"`
 	metadata.MetricsBuilderConfig           `mapstructure:",squash"`
-	Method                                  string `mapstructure:"method"`
```

Comment:
I know the component is alpha, but shouldn't we still avoid such interface breaking changes and support these options as deprecated for some time? cc @codeboten

Reply:
The component is still marked as development: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/httpcheckreceiver#http-check-receiver

Reply:
Also I see we don’t handle Exponential histograms yet, right?